contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -126,6 +137,22 @@ library ValidationLogic {
         uint256(DataTypes.InterestRateMode.STABLE) == interestRateMode,
       Errors.VL_INVALID_INTEREST_RATE_MODE_SELECTED
     );
+    
+    (vars.totalSupplyStableDebt, ) = IStableDebtToken(reserve.stableDebtTokenAddress)
+      .getTotalSupplyAndAvgRate();
+
+    vars.totalSupplyVariableDebt = IVariableDebtToken(reserve.variableDebtTokenAddress)
```

Comment:
use totalSupply(). We will refactor the code afterwards to fix #122 and replace this with a cached version